docs/getting-started.md

```diff
@@ -0,0 +1,30 @@
+---
+id: guides
+title: Guides
+sidebar_label: Guides
+---
+
+Getting started on Polkadot is easy! The wiki already contains guides for doing most of the core
+functionality.
+
+- [Creating an Account][account generation]
+- Sending a Transaction (Coming soon!)
+- [Staking as Nominator][nominator]
+- [Running a Validator][validator]
+- [Setting an Identity][identity]
+- [Creating a Proxy Account][proxy]
+- [Making Proposals and Voting for Referenda][democracy]
+- Social Recovery (Coming soon!)
+- [Running for the Council][council]
+- [Voting for Councillors][council voting]
+- [Using the Treasury][treasury]
+
+[account generation]: https://example.com
```

Comment:
This should be `<li>`s as well, no?

Reply:
No, this is Logan's fancy markdown way of keeping the actual hyperlink out of the text :P